tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/util/TlaExUtil.scala

```diff
@@ -21,6 +21,12 @@ object TlaExUtil {
       case OperEx(TlaActionOper.prime, NameEx(name)) =>
         used = used + (name + "'")
 
+      // Do not count the fold operator LET-IN itself
```

Comment:
shall we add a more general treatment for lambdas instead of the specialized code?

Reply:
Maybe, but not in this PR, this belongs to #872.